### ISSUE TITLE
MO 2.0.3a crashes on merging and replacing mods

### ISSUE THREAD

**Issue by striker873** [TanninOne/modorganizer#480]
Whenever I try to update a mod, merging a fix or an update or replacing the entire mod with a new versio MO crashes.

Alson overwrite system does not work with esp files for me, MO will not overwrite esps....

System: Windows 10 Pro x64 || i7-4970K 4.00GHz || 16,0 GB || Nvidia GeForce GTX 970

**Comment by LawLeewer** [comment]
I'm having the same trouble here.   It doesn't only crash if I update a mod, it also crashes if I try to install one.

**Comment by striker873** [comment]
Yup, on new mod install also crashes... but on restart the new mod shows...

Anyway.. thanks for this amazing app con think on playing without it

**Comment by TanninOne** [comment]
please try if deleting plugins/bsa_extractor.dll fixes the install-crash.

**Comment by DeadMor0z** [comment]
Removing bsa_extractor.dll fixes issue. Confirm.

**Comment by pressure68** [comment]
I can second confirmation. I was crashing after every mod install, aft deleting that file I installed several without issue.

**Comment by ogrotten** [comment]
Thirding confirmation renaming `bsa_extractor.dll` > `bsa_extractor.!dll` solves the crash.

BTW: It was crashing on any install for me, not just updates.

**Comment by Nyhkalas** [comment]
I am having this issue with "My HUD 3.12-1476-3-12.rar". Renaming bsa_extractor.dll > bsa_extractor.!dll fixed all my other mods from crashing except My HUD. 

[ModOrganizer.exe.dmp.zip](https://github.com/TanninOne/modorganizer/files/102017/ModOrganizer.exe.dmp.zip)

**Comment by TanninOne** [comment]
@Nyhkalas I believe the crash with MyHUD to be fixed in alpha4, can you please verify?
Btw.: This mod will not be successfully installed. The reason is that this installer is a scripted installer and those require NMM code in MO and the NMM guys haven't updated their public code repository since before fallout 4 came out.
tl;dr: I can't fix that.
